src/solvers/nonlinear_solid_solver.cpp

```diff
@@ -173,13 +173,13 @@ void NonlinearSolidSolver::completeSetup()
 
   // Set the MFEM abstract operators for use with the internal MFEM solvers
   if (timestepper_ == serac::TimestepMethod::QuasiStatic) {
-    solver_.solver().iterative_mode = true;
+    solver_.nonlinearSolver()->iterative_mode = true;
     nonlinear_oper_                 = std::make_unique<NonlinearSolidQuasiStaticOperator>(std::move(H_form));
     solver_.SetOperator(*nonlinear_oper_);
   } else {
-    solver_.solver().iterative_mode = false;
+    solver_.nonlinearSolver()->iterative_mode = false;
     timedep_oper_                   = std::make_unique<NonlinearSolidDynamicOperator>(
-        std::move(H_form), std::move(S_form), std::move(M_form), ess_bdr_, solver_.solver(), lin_params_);
+        std::move(H_form), std::move(S_form), std::move(M_form), ess_bdr_, *solver_.nonlinearSolver(), lin_params_);
```

Comment:
The reason we expose the solvers directly is mostly to set the iterative mode.  We could use the EquationSolver interface in the NonlinearSolidDynamicOperator if we expose the "getConverged".  Would it be reasonable to associate the timestep method with the EquationSolver object instead of the physics solver?  It could still be exposed via an accessor and would keep all equation solver configuration within the same object.